benchmarks/dynamo/common.py

```diff
@@ -3666,7 +3667,7 @@ def run(runner, args, original_dir=None):
     if args.ci:
         if args.accuracy:
             # Run fewer iterations when checking accuracy
-            args.repeat = 2
+            args.repeat = min(args.repeat, 2)
```

Comment:
How about the `--performance` mode?

Reply:
@desertfire I might misunderstand your comment, but this is the only place I found where we overwrite the repeat flag. It's not strictly necessary that I change this. I just thought it was weird if I call the subprocess with `--repeat=1` and this code would set it to 2.